### PR TITLE
mouse transition effect

### LINKED ISSUE #214: Mouse Transition Effect
### Describe the feature

I want to add a mouse transition effect on the main KWOC frontend. It will look like as the snow particles are falling on the main page, same small snow particles will be glow where ever mouse pointer will glow. Please assign me this issue so that i can start working on this.

### Why do you think this feature should be implemented?

_No response_

### Additional context

I am contributing under KWOC 2023

### Additional Information

- [X] I have used the search function to check if an issue already exists
- [X] I'd like to work on this issue

### PR DESCRIPTION
**Pull request by MitulSonagara**
# Title and Issue number 
Title : Mouse Transition Effect

Issue No. : #214 

Close #214

**Additional context (Mandatory )**
I have created the mouse transition effect as per demo given in my issue detail... please review this and give me further improvements.

### CODE REVIEW

**Comment by Nafisay** [comment]
@MitulSonagara as you can see that the snowflakes have a dark border which does not match the snowflakes of the landing page. Please make the required changes so as they match the theme

**Comment by Nafisay** [comment]
Also can you make the number of snowflakes more sparse and slower. I think it will look good that way

**Comment by MitulSonagara** [comment]
Sure, I can try that way also...

**Comment by MitulSonagara** [comment]
Can you please provide me that the snow flakes are images or any emoji...

**Comment by Nafisay** [comment]
You will find the snowflakes svg file here
[https://github.com/kossiitkgp/KWoC-Frontend/tree/truth-redefined-again/src/assets](url)

Note that there are 3 types of snowflakes involved. Try to incorporate all of them

**Comment by MitulSonagara** [comment]
@Nafisay I have updated the code. But i can not see the changes on the deployment but i can see this on my local server... can you please check it.

**Comment by Nafisay** [comment]
Same, even I can't see them

**Comment by MitulSonagara** [comment]
I'll inform you after some changes...

**Comment by Nafisay** [comment]
Sorry I can't see any changes

**Comment by harshkhandeparkar** [comment]
@MitulSonagara were you able to debug the issue?

File: index.html [hunk 1]
```diff
@@ -1,33 +1,111 @@
...
+  <meta property="og:site_name" content="Kharagpur Winter of Code 2023" />
+  <meta name="twitter:image:alt" content="Kharagpur Winter of Code logo" />
+
+  <title>Kharagpur Winter of Code | For the love of Open Source</title>
+</head>
+
+<body>
+  <div id="root"></div>
+  <script type="module" src="/src/main.tsx"></script>
+  <script>
```
Comment (harshkhandeparkar) [inline index.html:30]: Please do not add javascript directly. Use react.

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]

**Comment by harshkhandeparkar** [comment]
If you are unable to debug this, I would suggest closing this PR since it is not very important. Also adding transition effects linked to the mouse might lead to performance issues (we had removed some effects due to this earlier).

**Pull request closed without merging**